tst/testinstall/opers/IsSolvableGroup.tst

```diff
@@ -0,0 +1,50 @@
+gap> START_TEST("IsSolvableGroup.tst");
+gap> List(AllGroups(120), IsSolvableGroup);
+[ true, true, true, true, false, true, true, true, true, true, true, true, 
+  true, true, true, true, true, true, true, true, true, true, true, true, 
+  true, true, true, true, true, true, true, true, true, false, false, true, 
+  true, true, true, true, true, true, true, true, true, true, true ]
+gap> List(AllTransitiveGroups(DegreeAction, 8), IsSolvable);
+[ true, true, true, true, true, true, true, true, true, true, true, true, 
+  true, true, true, true, true, true, true, true, true, true, true, true, 
+  true, true, true, true, true, true, true, true, true, true, true, true, 
+  false, true, true, true, true, true, false, true, true, true, true, false, 
+  false, false ]
+gap> IsSolvable(DihedralGroup(24));
+true
+gap> IsSolvable(DihedralGroup(IsFpGroup,24));
+true
+gap> DerivedSeries(Group(()));
+[ Group(()) ]
+gap> G := Group((6,7,8,9,10),(8,9,10),(1,2)(6,7),(1,2,3,4)(6,7,8,9));;
+gap> Length(DerivedSeriesOfGroup(G));
+4
+gap> HasIsSolvableGroup(G) and not IsSolvable(G) and HasIsAbelian(G) and not IsAbelian(G);
+true
+gap> IsSolvableGroup(AbelianGroup([2,3,4,5,6,7,8,9,10]));
+true
+gap> HasIsSolvableGroup(AbelianGroup(IsFpGroup,[2,3,4,5,6,7,8,9,10]));
+true
+gap> IsSolvableGroup(AbelianGroup(IsFpGroup,[2,3,4,5,6,7,8,9,10]));
+true
+gap> IsSolvableGroup(Group(()));
+true
+gap> A := AbelianGroup([3,3,3]);; H := AutomorphismGroup(A);;
+gap> B := SylowSubgroup(H, 13);; G := SemidirectProduct(B, A);;
+gap> HasIsSolvableGroup(G) and IsSolvable(G);
+true
+gap> F := FreeGroup("r", "s");; r := F.1;; s := F.2;;
+gap> G := F/[s^2, s*r*s*r];;
+gap> IsSolvable(G) and HasIsAbelian(G) and not IsAbelian(G);
```

Comment:
You are right that if a new method comes, then these tests may fail. However, after I checked the examples, I am now pretty confident that I wanted to check whether the method in this PR adds the appropriate ````IsSolvable```` and ````IsAbelian```` flags. The ````DerivedSeriesOfGroup```` should not (and does not) add most of these flags, mainly because it stops due to empty ````AbelianInvariants````, instead of knowing that the last entry is trivial. 

I would rather prefer to write a comment into the test file about these examples explaining this situation. But of course if you still prefer otherwise, I can run ````DerivedSeriesOfGroup```` first, and then check for the necessary flags, but that would not test whether the ````IsSolvable```` method in this PR really adds the appropriate flags.